.github/workflows/test.yml

```diff
@@ -69,7 +69,7 @@ jobs:
           path: node_modules/.cache
           key: ${{ runner.os }}-jest-${{ github.run_id }}
           restore-keys: ${{ runner.os }}-jest-
-      - run: yarn test --coverage --maxWorkers=100%
+      - run: yarn test --coverage --maxWorkers=2
```

Comment:
`100%` should be equivalent to `2`, and is more future-proof. Can this line be reverted, or is there a reason they are not equivalent?

Reply:
i'll revert to compare the run times

Reply:
looks similar - a lot of the difference is from caching, which should be discounted.

Reply:
yep, that sounds right. the actual unit test step is taking the same amount of time